recipe/meta.yaml

```diff
@@ -1,43 +1,93 @@
-  url: https://pypi.io/packages/source/e/esda/esda-{{ version }}.tar.gz
-  sha256: 308b43cdc2f7af9b01c4a71fb1869092d48bcdf953e99dba8fc8239745779f1e
+  url: https://github.com/pysal/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 8557fd5da1d7ee23cdcc8cb8f83ab8e772f7200b647303f3b0bb8836ae19fe9e
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # Need to avoid error: setuptools-scm was unable to detect version !!!
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  # scipy >=1.9,<1.12 is not avaliable on py313
+  skip: True  # [py<38 or py>312]
 
 requirements:
   host:
     - python
+    - wheel
+    - setuptools >=61.0
+    - setuptools_scm >=6.2
     - pip
   run:
-    - python >3.4
+    - python
     - libpysal
-    - pandas
-    - scikit-learn
-    - scipy >=0.11
+    - pandas >1.4
+    - scikit-learn >=1.0
+    # Build error appears:
+    # E cannot import name 'inf' from 'scipy'
+    # The issue is likely due to the fact that Scipy removed scipy.inf in recent versions (starting from 1.13).
+    # Changed upstream scipy >=1.9 on >=1.9,<1.12
+    - scipy >=1.9,<1.12
+
+# Tests fails on :
+# - py>39
+#     E   FloatingPointError: invalid value encountered in voronoi_polygons
+# - py<=39
+#     TypeError: float() argument must be a string or a number, not 'Point'
+{% set tests_to_skip = "test_isolation_options" %}
+{% set tests_to_skip = tests_to_skip + " or test_isolation_valid" %}
+{% set tests_to_skip = tests_to_skip + " or test_prominence_options" %}
+{% set tests_to_skip = tests_to_skip + " or test_prominence_valid" %}
+{% set tests_to_skip = tests_to_skip + " or test_to_elevation" %}
+
+
+# E to_adjlist() got an unexpected keyword argument 'drop_islands'
+{% set tests_to_skip = tests_to_skip + " or test_silhouette_alist" %}  # [py<=39]
+{% set tests_to_skip = tests_to_skip + " or test_boundary" %}  # [py<=39]
+{% set tests_to_skip = tests_to_skip + " or test_local_geary" %}  # [py<=39]
+{% set tests_to_skip = tests_to_skip + " or test_Local_Join_Counts_MV" %}  # [py<=39]
+{% set tests_to_skip = tests_to_skip + " or test_Local_Join_Counts_BV" %}  # [py<=39]
+{% set tests_to_skip = tests_to_skip + " or test_Join_Counts_Locals" %}  # [py<=39]
+{% set tests_to_skip = tests_to_skip + " or test_by_col" %}  # [py<=39]
+{% set tests_to_skip = tests_to_skip + " or test_Join_Counts" %}  # [py<=39]
 
 test:
+  source_files:
+    - esda
   imports:
     - esda
   commands:
     - pip check
+    - pytest -v -k "not ({{ tests_to_skip }})" esda/tests  # [not (linux and s390x)]
   requires:
     - pip
-    - python <3.10
+    - pytest
+    - geopandas >=0.7.0  # [not (linux and s390x)]
+    # E AttributeError: module 'fiona' has no attribute 'path'
+    # So upgrading geopandas to 0.14.4 should fix it.
+    # But latest geopandas we have is 0.14.2, so we need to forcing fiona to stay on version <1.10 should also work.
```

Comment:
@Andrii-Osipov `esda` is part of the `geopandas 1.0.1` alignement, you find it at https://github.com/AnacondaRecipes/geopandas-feedstock/pull/11